### PR TITLE
add new gpg key for rvm

### PR DESCRIPTION
**Pull request by cabrinha**
```
==> default: Notice: /Stage[main]/Rvm::Gnupg_key/Gnupg_key[rvm_D39DC0E3]/ensure: created
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: Downloading https://github.com/rvm/rvm/archive/1.29.8.tar.gz
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: Downloading https://github.com/rvm/rvm/releases/download/1.29.8/1.29.8.tar.gz.asc
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: gpg: Signature made Wed May  8 14:14:49 2019 UTC
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: gpg:                using RSA key 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: gpg: Can't check signature: No public key
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: GPG signature verification failed for '/usr/local/rvm/archives/rvm-1.29.8.tgz' - 'https://github.com/rvm/rvm/releases/download/1.29.8/1.29.8.tar.gz.asc'! Try to install GPG v2 and then fetch the public key:
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: 
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns:     sudo gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: 
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: or if it fails:
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: 
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns:     command curl -sSL https://rvm.io/mpapis.asc | sudo gpg --import -
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns:     command curl -sSL https://rvm.io/pkuczynski.asc | sudo gpg --import -
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: 
==> default: Notice: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: In case of further problems with validation please refer to https://rvm.io/rvm/security
==> default: Error: 'curl -fsSL https://get.rvm.io | bash -s -- --version latest' returned 2 instead of one of [0]
==> default: Error: /Stage[main]/Rvm::System/Exec[system-rvm]/returns: change from 'notrun' to ['0'] failed: 'curl -fsSL https://get.rvm.io | bash -s -- --version latest' returned 2 instead of one of [0]
```

### CODE REVIEW

**Comment by benh57** [comment]
+1 this is required, or this module totally fails to function.

**Comment by cabrinha** [comment]
bump

**Comment by mmoll** [comment]
@janschumann ping?

File: manifests/gnupg_key.pp [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # RVM's GPG key import
-
 class rvm::gnupg_key(
-  $key_id = $rvm::params::gnupg_key_id,
-  $key_server = $rvm::params::key_server) inherits rvm::params {
```
Comment (alexjfisher) [inline manifests/gnupg_key.pp:5]: The `rvm::system` class is still expecting a `key_server` parameter.
https://github.com/janschumann/puppet-rvm/blob/ba09a82337104d1961014bb6dd89046e3e422ffa/manifests/system.pp#L41

Comment (cabrinha) [reply manifests/gnupg_key.pp:5]: I pushed a new commit. Not sure if it was correct.

**Comment by dhs-rec** [comment]
@janschumann, I know this is unrelated but could you please enable issues for this repo?

**Pull request closed without merging**